project/DistributionPackage.scala

```diff
@@ -180,6 +170,40 @@ object DistributionPackage {
       graalVersion = graalVersion,
       javaVersion  = javaVersion
     )
+
+    indexStdLib(
+      stdLibVersion = targetStdlibVersion,
+      ensoVersion   = ensoVersion,
+      stdLibRoot    = distributionRoot / "lib",
+      ensoExecutable =
+        distributionRoot / "bin" / Platform.executableFileName("enso"),
+      log = log
+    )
+  }
+
+  def indexStdLib(
+    stdLibVersion: String,
+    ensoVersion: String,
+    stdLibRoot: File,
+    ensoExecutable: File,
+    log: Logger
+  ): Unit = {
+    for {
+      libMajor <- stdLibRoot.listFiles()
+      libName  <- (stdLibRoot / libMajor.getName).listFiles()
+    } yield {
+      val command = Seq(
```

Comment:
This needs to be guarded since we certainly don't want to regenerate those on every rebuild of the engine. It simply takes too long.
At the same time it is currently a requirement for the suggestions to be already serialized.

So probably some timestamping is necessary.

Reply:
Another problem is that every execution of `built-distribution/enso-engine-*/*/bin/enso --no-compile-dependencies --compile built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/lib/Standard/*/0.0.0-dev` is building suggestions for `Standard.Table.Data.Table`. Even when processing base or geo library. Not sure why, yet.